tools/target-server-validator/callout/src/main/java/PortOpenCheck.java

```diff
@@ -72,12 +95,34 @@ private static String available(final String host, final int port) {
   public ExecutionResult execute(final MessageContext messageContext,
     final ExecutionContext executionContext) {
     try {
-      String hostname = messageContext.getMessage().getHeader("host_name");
-      String port = messageContext.getMessage().getHeader("port_number");
-      int portnumber = Integer.parseInt(port);
-      String status = available(hostname, portnumber);
-      messageContext.setVariable("flow.reachableStatus", status);
-      return ExecutionResult.SUCCESS;
+      String payload = (String) messageContext.getVariable("request.content");
```

Comment:
Quite a complex datastructure in the request. Can we add some docs on the stucture of the payload?

Reply:
+1 please update the readme to include example JSON for both the request and response

Reply:
Updated the README.